src/options/modules/ColorSchemeModeHelper.js

```diff
@@ -0,0 +1,16 @@
+/* this function changes screenshot in the options menu as per user's OS theme.By defualt 
+a user will see light themed screenshot but if the user has enabled dark theme
+in their OS the they would see dark-themed screenshot. It makes use of the following CSS media query
+and detects its change of state by attaching a js listener to it.
+ */
+const dark = window.matchMedia("(prefers-color-scheme: dark)");
+function changeScreenshotTheme(darkQuery){
```

Comment:
BTW maybe add a link to https://bugzilla.mozilla.org/show_bug.cgi?id=1595037 as a comment to make sure we do not forget this does not work.

Something like:
```suggestion
// does not work in Firefox currently due to https://bugzilla.mozilla.org/show_bug.cgi?id=1595037
function changeScreenshotTheme(darkQuery){
```

(maybe better integrate it into the JSDoc though)